CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 Change Log
 ==========
+2.0.0
+-----
+
+* Upgraded to Python 3.
+* Increased timeout to 6 seconds.
```

Comment:
Out of the context of the commit this description is a little vague, maybe: "Increased `urlopen` timeout to 6 seconds when determining file system location."